omc3/plotting/plot_checked_corrections.py

```diff
@@ -403,6 +429,72 @@ def save_plots(output_dir, figure_dict, input_dir=None):
             fig.savefig(output_path)
 
 
+def show_plots(figure_dict: Dict[str, Figure]):
+    """ Shows plots.
+    If qtpy is installed, they are shown in a single window.
+    The individual corrections are sorted in to vertical tabs,
+    the optics parameter into horizontal tabs. 
+    If qtpy is not installed, they are simply shown as individual figures.
+    This is not recommended
+    """
+    try:
+        window = VerticalTabWindow("Correction Check")
+    except TypeError:
+        log_no_qtpy_many_windows()
+        for name, fig in figure_dict.items():
+            create_pyplot_window_from_fig(fig, title=name.replace(SPLIT_ID, " "))
+        plt.show()
+        return
+
+    rdt_pattern = re.compile(r"f\d{4}")
+    rdt_complement = {
+        RDT_REAL_COLUMN.text_label: RDT_IMAG_COLUMN,
+        RDT_AMPLITUDE_COLUMN.text_label: RDT_PHASE_COLUMN,
+    }
+
+    correction_names = sorted(set([n.split(SPLIT_ID)[0] for n in figure_dict.keys() if SPLIT_ID in n]))
+    for correction_name in [None] + list(correction_names):
+        if not correction_name:
+            parameter_names = iter(sorted(n for n in figure_dict.keys() if SPLIT_ID not in n))
+            correction_tab_name = "All Corrections"
+        else:
+            parameter_names = iter(sorted(n for n in figure_dict.keys() if correction_name in n))
+            correction_tab_name = correction_name
+
+        current_tab = TabWidget(title=correction_tab_name)
+        window.add_tab(current_tab)
+
+        for name_x in parameter_names:
+            tab_prename = name_x.split(SPLIT_ID)[-1]
+
+            if rdt_pattern.match(tab_prename):
+                rdt, column = tab_prename.split("_")[:2]
+                try:
+                    complement_column: ColumnsAndLabels = rdt_complement[column]
+                except KeyError:
+                    # skip phase and imag as they are name_y for amp and real.
+                    continue
+
+                if not correction_name:
+                    name_y = "_".join([rdt, complement_column.text_label, complement_column.expected_column])
+                else:
+                    name_y = "_".join(name_x.split("_")[:-1] + [complement_column.text_label,])
+
+                tab_name = f"{rdt} {column[0].upper()}{complement_column.text_label[0].upper()}"
+
+            else:
+                tab_name = " ".join(tab_prename.split("_")[:-1 if correction_name else -2])
+                name_y = next(parameter_names)
+
+            new_tab = PlotWidget(
```

Comment:
I'm confused why `new_tab` is not a `TabWidget`?

Reply:
because the TabWidget is a container for tabs. But each tab itself is just a widget.